inspire/base/templates/format/record/Inspire_Default_HTML_detailed.tpl

```diff
@@ -73,4 +72,14 @@
       {{ record_citations() }}
     </div>
   </div>
+  <script type="text/javascript">
```

Comment:
@jmartinm placing this in the `javascript` or `bundles` block doesn't seem to work. This template inherits from https://github.com/inveniosoftware/invenio-formatter/blob/master/invenio_formatter/templates/format/record/Default_HTML_detailed.tpl , which in turn inherits from https://github.com/inveniosoftware/invenio-formatter/blob/master/invenio_formatter/templates/format/record/Default_HTML_detailed_base.tpl. Neither of these have a `javascript` or `bundles` block.